WordPress/src/main/java/org/wordpress/android/ui/posts/PostUtils.java

```diff
@@ -229,7 +229,7 @@ public static boolean postHasEdits(@Nullable PostImmutableModel oldPost, PostImm
                                     && newPost.getTagNameList().containsAll(oldPost.getTagNameList())
                                     && oldPost.getCategoryIdList().containsAll(newPost.getCategoryIdList())
                                     && newPost.getCategoryIdList().containsAll(oldPost.getCategoryIdList())
-                                    && PostLocation.equals(oldPost.getLocation(), newPost.getLocation())
```

Comment:
Shouldn't we call PostLocation.companion.equals here?

Reply:
you're right 👍